dpdispatcher/batch_object.py

```diff
@@ -9,6 +9,9 @@
 from dpdispatcher.ssh_context import SSHContext
 from dpdispatcher.submission import Resources
 
+# from typing import TypedDict
+from typing_extensions import TypedDict
```

Comment:
this is a third-party module so you need to add it to setup.py

Reply:
added

Reply:
It looks like I use syntax supported only in python 3.9.
I will fix it